index.html

```diff
@@ -291,6 +292,8 @@ <h2>Contact me</h2>
 				<button type="submit" id="submit-btn">Get In Touch</button>
 			</form>
 		</section>
-			<script src="index.js"></script>
+		<section class="modals"></section>
```

Comment:
- [ ] Kindly please remove the transparency in your modals, in order to follow the Figma template.  It's also really difficult to read.
- [ ] In the mobile view, please kindly justify and center the buttons horizontally, as shown on the Template.

Look: 

Your implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/152293272-08f3af5e-ff5b-4160-b214-5c9f63d731e2.png) | ![image](https://user-images.githubusercontent.com/57421823/152293326-81350dbf-8ce0-495e-b0f3-ffbb5bfb2d0d.png)

That way, your popups will look strikingly beautiful 💪